app/core/interfaces/textinput/index.js

```diff
@@ -71,22 +71,22 @@ define(['app', 'core/UIComponent', 'core/UIView', 'core/t'], function(app, UICom
     // Validate String  Checks the passed in value against validation_string
     // @param e : keypress event object
     validateString: function(e) {
-      var validationMessage = this.options.settings.get('validation_message') || app.DEFAULT_VALIDATION_MESSAGE;
+      var validationMessage = this.options.settings.get('validation_message') || __t('confirm_invalid_value');
       var chars;
       switch(this.options.settings.get('validation_type')) {
         case ('bl') :
-          chars = this.options.settings.get('validation_string').split("");
-          return $.inArray(String.fromCharCode(e.which), chars) === -1;
+          chars = this.options.settings.get('validation_string').split('');
+          return !chars.includes(String.fromCharCode(e.which));
```

Comment:
(It was a usage of `Array.prototype.includes` instead of the matching String version, but the browser support wasn't ideal nonetheless)